### PR TITLE
Merging template updates 2.3.2

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/branch.yml [hunk 1]
```diff
@@ -43,3 +42,4 @@ jobs:
             Thanks again for your contribution!
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           allow-repeats: false
+#
```
Comment (maxulysse) [inline .github/workflows/branch.yml:45]: where is this coming from?

File: .github/workflows/linting.yml [hunk 2]
```diff
@@ -138,3 +78,5 @@ jobs:
             lint_log.txt
             lint_results.md
             PR_number.txt
+
+#
```
Comment (maxulysse) [inline .github/workflows/linting.yml:82]: ?

Comment (FriederikeHanssen) [reply .github/workflows/linting.yml:82]: Template thats why I left it in 🤷 I thought there was a PR to remove trailing #, need to investigate

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# ![nf-core/sarek](docs/images/nf-core-sarek_logo_light.png#gh-light-mode-only) ![nf-core/sarek](docs/images/nf-core-sarek_logo_dark.png#gh-dark-mode-only)
+# ![nf-core/sarek](docs/images/nf-core/sarek_logo_light.png#gh-light-mode-only) ![nf-core/sarek](docs/images/nf-core/sarek_logo_dark.png#gh-dark-mode-only)
```
Comment (maxulysse) [inline README.md:1]:
```suggestion
# ![nf-core/sarek](docs/images/nf-core-sarek_logo_light.png#gh-light-mode-only) ![nf-core/sarek](docs/images/nf-core-sarek_logo_dark.png#gh-dark-mode-only)
```

I don't think the path has changed

File: assets/multiqc_config.yaml [hunk 4]
```diff
@@ -1,34 +1,34 @@
 custom_logo: ../../../docs/images/nf-core_sarek_logo.png
 custom_logo_url: https://github.com/nf-core/sarek/
```
Comment (maxulysse) [inline assets/multiqc_config.yaml:2]: I think that the content of this file can be dumped into the newly created `multiqc_config.yml` that should replace this one

**Review by maxulysse — CHANGES_REQUESTED** [review]
just need some changes in the readme and the multiqc_config.yml file

**Review by maxulysse — APPROVED** [review]
<3

**Comment by maxulysse** [comment]
`nf-core lint` is failing, but not sure about the issue there

**Comment by FriederikeHanssen** [comment]
Running it locally I get a different error with `nf-core lint`:

```
╭──────────────────────────────────────────┬──────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Module name                              │ File path                                                    │ Test message                                                                                                                                                                                                                                                 │
├──────────────────────────────────────────┼──────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ ascat                                    │ modules/nf-core/modules/ascat/meta.yml                       │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ controlfreec                             │ modules/nf-core/modules/controlfreec/meta.yml                │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ deeptools/bamcoverage                    │ modules/nf-core/modules/deeptools/bamcoverage/meta.yml       │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ gatk4/gatherpileupsummaries              │ modules/nf-core/modules/gatk4/gatherpileupsummaries/meta.yml │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ gatk4/getpileupsummaries                 │ modules/nf-core/modules/gatk4/getpileupsummaries/meta.yml    │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ gatk4/variantrecalibrator                │ modules/nf-core/modules/gatk4/variantrecalibrator/meta.yml   │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ msisensorpro/msi_somatic                 │ modules/nf-core/modules/msisensorpro/msi_somatic/meta.yml    │ Local copy of module does not match remote                                                                                                                                                                                                                   │
│ msisensorpro/scan                        │ modules/nf-core/modules/msisensorpro/scan/meta.yml           │ Local copy of module does not match remote                                                                                                                                                                                                                   │
╰──────────────────────────────────────────┴──────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭───────────────────────╮
│ LINT RESULTS SUMMARY  │
├───────────────────────┤
│ [✔] 371 Tests Passed  │
│ [?]   8 Tests Ignored │
│ [!]  15 Test Warnings │
│ [✗]   8 Tests Failed  │
╰───────────────────────╯
```

**Comment by maxulysse** [comment]
Manage to run the `nf-core lint` locally on @FriederikeHanssen branch:
```
╰─≻nf-core lint                                                                           

                                          ,--./,-.
          ___     __   __   __   ___     /,-._.--~\
    |\ | |__  __ /  ` /  \ |__) |__         }  {
    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                          `._,._,'

    nf-core/tools version 2.3.2 - https://nf-co.re


INFO     Testing pipeline: .                                                                                                                                                                                                   __init__.py:243
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ General lint results                                                                                                                                                                                                                       │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [?] 8 Tests Ignored                                                                                                                                                                                                                        │
├────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ files_unchanged: File ignored due to lint config: .github/PULL_REQUEST_TEMPLATE.md                                                                                                                                                         │
│ files_unchanged: File ignored due to lint config: .github/workflows/branch.yml                                                                                                                                                             │
│ files_unchanged: File ignored due to lint config: .github/workflows/linting_comment.yml                                                                                                                                                    │
│ files_unchanged: File ignored due to lint config: .github/workflows/linting.yml                                                                                                                                                            │
│ files_unchanged: File ignored due to lint config: assets/nf-core-sarek_logo_light.png                                                                                                                                                      │
│ files_unchanged: File ignored due to lint config: docs/images/nf-core-sarek_logo_light.png                                                                                                                                                 │
│ files_unchanged: File ignored due to lint config: docs/images/nf-core-sarek_logo_dark.png                                                                                                                                                  │
│ files_unchanged: File ignored due to lint config: lib/NfcoreTemplate.groovy                                                                                                                                                                │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [!] 7 Test Warnings                                                                                                                                                                                                                        │
├────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ pipeline_todos: TODO string in awsfulltest.yml: You can customise AWS full pipeline tests as required                                                                                                                                      │
│ pipeline_todos: TODO string in test_full.config: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)                                                                                │
│ pipeline_todos: TODO string in test_full.config: Give any required params for the test so that command line flags are not needed                                                                                                           │
│ pipeline_todos: TODO string in test_full.config: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)                                                                                │
│ pipeline_todos: TODO string in test_full.config: Give any required params for the test so that command line flags are not needed                                                                                                           │
│ schema_description: No description provided in schema for parameter: umi_read_structure                                                                                                                                                    │
│ schema_description: No description provided in schema for parameter: group_by_umi_strategy                                                                                                                                                 │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Module lint results                                                                                                                                                                                                                        │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [!] 2 Test Warnings                                                                                                                                                                                                                        │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭──────────────────────────────────────────┬───────────────────────────────────────────────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Module name                              │ File path                                     │ Test message                                                                                                                                    │
├──────────────────────────────────────────┼───────────────────────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ deeptools/bamcoverage                    │ modules/nf-core/modules/deeptools/bamcoverage │ New version available                                                                                                                           │
│ samtools/mpileup                         │ modules/nf-core/modules/samtools/mpileup      │ New version available                                                                                                                           │
╰──────────────────────────────────────────┴───────────────────────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭───────────────────────╮
│ LINT RESULTS SUMMARY  │
├───────────────────────┤
│ [✔] 385 Tests Passed  │
│ [?]   8 Tests Ignored │
│ [!]   9 Test Warnings │
│ [✗]   0 Tests Failed  │
╰───────────────────────╯
```
I have no idea what is the issue...

**Comment by FriederikeHanssen** [comment]
yep, also not sure why Markdownlint and YAML is still not running 😞

**Comment by ewels** [comment]
> yep, also not sure why Markdownlint and YAML is still not running 😞

These no longer exist after the latest pipeline template sync. They must be set as required tests in the repo settings for the protected branch, which is why they show up as _"Expected"_

**Comment by maxulysse** [comment]
> > yep, also not sure why Markdownlint and YAML is still not running disappointed
> 
> These no longer exist after the latest pipeline template sync. They must be set as required tests in the repo settings for the protected branch, which is why they show up as _"Expected"_

good catch, I'll remove them right away

**Comment by ewels** [comment]
Just done it :)

**Comment by maxulysse** [comment]
> Just done it :)

Saw it, but I added Prettier

**Pull request merged**